_topic_map.yml

```diff
@@ -183,6 +183,8 @@ Topics:
 - Name: Installing on GCP
   Dir: installing_gcp
   Topics:
+  - Name: Preparing to install on GCP
```

Comment:
This title is [already in the topic map in the main openshift-docs master branch](https://docs.openshift.com/container-platform/4.8/installing/installing_gcp/preparing-to-install-on-gcp.html). I'm a bit confused as to why this is showing up as an addition.

Reply:
I think I need to actually delete the topic_map.yml from my commends, just as I did for the GCP document. My commends keep getting that file also and re-adding it.

Reply:
"git rm _topic_map.yml" before this commend.